package.json

```
@@ -108,7 +108,7 @@
     "language-text": "https://www.atom.io/api/packages/language-text/versions/0.7.4/tarball",
     "language-todo": "https://www.atom.io/api/packages/language-todo/versions/0.29.4/tarball",
     "language-toml": "https://www.atom.io/api/packages/language-toml/versions/0.20.0/tarball",
-    "language-typescript": "https://www.atom.io/api/packages/language-typescript/versions/0.5.3/tarball",
+    "language-typescript": "https://www.atom.io/api/packages/language-typescript/versions/0.5.2/tarball",
```

Oh, weird. I wonder how this Typescript change got in here.

@lee-dohm: I see a few recent PRs related to upgrading language-typescript (https://github.com/atom/atom/pull/19951, https://github.com/atom/atom/pull/19952, and https://github.com/atom/atom/pull/19953). It looks like the master branch currently has conflicting version info for language-typescript:

https://github.com/atom/atom/blob/70b1b947e947059eb3c5c21af0abb1292a5e33d6/package.json#L263

https://github.com/atom/atom/blob/70b1b947e947059eb3c5c21af0abb1292a5e33d6/package.json#L111

Can you tell me which version we should be using at this point?

---

We should be using v0.5.3. I forget why I had to revert the bump to v0.5.3 and then redo it though 🤔

---

Oh, it was because I didn't include the `package-lock.json` change in the first PR, so the build failed.

---

Thanks for that context, @lee-dohm. We'll remove the language-typescript bits from this pull request, and we'll open a separate pull request to address the conflicting language-typescript versions.


```suggestion
    "language-typescript": "https://www.atom.io/api/packages/language-typescript/versions/0.5.3/tarball",
```

---

> We'll remove the language-typescript bits from this pull request, and we'll open a separate pull request to address the conflicting language-typescript versions.

I've opened https://github.com/atom/atom/pull/20044 to address the conflicting language-typescript versions.